tests/sentry/api/serializers/test_event.py

```diff
@@ -370,3 +375,39 @@ def test_ignores_patch(self, mock_index_state):
 
         result = serialize(event, None, DetailedEventSerializer())
         assert result["sdkUpdates"] == []
+
+    @override_options({"store.use-ingest-performance-detection-only": 1.0})
+    @override_options({"performance.issues.all.problem-creation": 1.0})
+    @override_options({"performance.issues.all.problem-detection": 1.0})
+    def test_performance_problem(self):
+        self.project.update_option("sentry:performance_issue_creation_rate", 1.0)
+        with mock.patch("sentry_sdk.tracing.Span.containing_transaction"), self.feature(
+            "projects:performance-suspect-spans-ingestion"
+        ):
+            manager = EventManager(make_event(**EVENTS["n-plus-one-in-django-index-view"]))
+            manager.normalize()
+            event = manager.save(self.project.id)
+        group_event = event.for_group(event.groups[0])
+
+        result = json.loads(json.dumps(serialize(group_event, None, DetailedEventSerializer())))
+        assert result["perfProblem"] == {
+            "cause_span_ids": ["9179e43ae844b174"],
+            "desc": "SELECT `books_author`.`id`, `books_author`.`name` FROM "
+            "`books_author` WHERE `books_author`.`id` = %s LIMIT 21",
+            "fingerprint": "19e15e0444e0bc1d5159fb07cd4bd2eb",
+            "offender_span_ids": [
```

Comment:
Should the output keys use `camelBack` casing? Most of the API uses `camelBack`.